### PR TITLE
Fixed: PulpTemporaryFile stored in the wrong location

### LINKED ISSUE #7319: Errors in "Contributing with code" documentation.
There are some logical errors in the quickstart documentation as well as incorrect links to further docs.

I'll provide a PR on this shortly.

### PR DESCRIPTION
**Pull request by fao89**
https://pulp.plan.io/issues/7319
closes #7319

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Comment by fao89** [comment]
We already merged:
- https://github.com/pulp/pulpcore/pull/859
- https://github.com/pulp/pulpcore/pull/860

With this PR, and maybe with https://pulp.plan.io/issues/7329
Should we release 3.6.1?

**Comment by mdellweg** [comment]
As much as i believe this is doing the trick, would you mind adding a unittest for it? I think, you can almost use the example from the ticket (creating a PTFile, checking it's path, try reading it).

**Review by ipanova — APPROVED** [review]

**Comment by ipanova** [comment]
@fao89  can you update the reference to the Artifactfilefield in the doc blocks?

File: pulpcore/tests/unit/models/test_content.py [hunk 1]
```diff
@@ -41,3 +41,21 @@ def test_remove_content(self):
         # Assumes creation is tested by test_create_and_read_content function
         Content.objects.filter(pk=content.pk).delete()
         self.assertFalse(Content.objects.filter(pk=content.pk).exists())
+
+
+class PulpTemporaryFileTestCase(TestCase):
+    def test_storage_location(self):
+        with tempfile.NamedTemporaryFile("ab") as tf:
```
Comment (mdellweg) [inline pulpcore/tests/unit/models/test_content.py:48]:
```suggestion
        if settings.DEFAULT_FILE_STORAGE != "pulpcore.app.models.storage.FileSystem":
            self.skipTest("Skipping test for nonlocal storage.")
        with tempfile.NamedTemporaryFile("ab") as tf:
```

**Review by mdellweg — APPROVED** [review]
Thank you for the tests!
I added a suggestion to conditionally skip the test. wdyt?

**Pull request merged**